src/extension.ts

```diff
@@ -149,12 +110,15 @@ export async function activateInternal(context: vscode.ExtensionContext, perfSta
                 azureResourceProviderManager,
                 azureResourceBranchDataProviderManager,
                 workspaceResourceProviderManager,
-                workspaceResourceBranchDataProviderManager);
+                workspaceResourceBranchDataProviderManager
+            );
         }
 
         return v2Api;
     }
 
+    v2ApiFactory();
```

Comment:
It feels like this method should have a return.

Reply:
Technically this isn't used yet. And later it will be `ext.v2.api = vsApiFactory();`. I'll remove it for now and add that stuff later when it's actually used.